app/src/main/java/org/oppia/app/player/audio/AudioViewModel.kt

```diff
@@ -114,4 +169,13 @@ class AudioViewModel @Inject constructor(
   private fun voiceOverToUri(voiceover: Voiceover?): String {
     return "https://storage.googleapis.com/$gcsResource/exploration/$explorationId/assets/audio/${voiceover?.fileName}"
   }
+
+  private fun <T> LiveData<T>.observeOnce(lifecycleOwner: LifecycleOwner, observer: Observer<T>) {
```

Comment:
Can we save this extension function in a Kotlin file somewhere rather than hiding it in a class? I think it's really useful and have found myself wanting other times.